Header header
int32[] contour
PushAction ann_push
PushAction polyreg_push
PushAction maxdist_push
PushAction average_push
TapAction ann_tap
TapAction polyreg_tap
TapAction maxdist_tap
TapAction average_tap